docs/readmes/lte/upgrade_1_6.md

```diff
@@ -0,0 +1,37 @@
+---
+id: upgrade_1_6
+title: Upgrade to v1.6
+hide_title: true
+---
+
+# Upgrade to v1.6
+
+### Fresh Install Notes
+
+For fresh gateway installs, please use the following install scripts.
+
+```bash
+wget https://raw.githubusercontent.com/magma/magma/master/lte/gateway/deploy/agw_install_ubuntu.sh
+chmod +x agw_install_ubuntu.sh
+./agw_install_ubuntu.sh
+```
+
+### Upgrading from previous releases
+
+Beginning with Magma Fuji (v1.5), Magma artifacts are now hosted on the new
```

Comment:
Nit: don't hard-break your lines. Each paragraph should be 1 line

Ref: https://magma.github.io/magma/docs/next/contributing/contribute_conventions#markdown

Reply:
Nit: I think the focus should be on the numeric version, rather than the human-readable name
```suggestion
Beginning with release v1.5 (Fuji), Magma artifacts are now hosted on the new
```